requirements.txt

```diff
@@ -1,8 +1,8 @@
-aiohttp[speedups]==3.8.1
+aiohttp[speedups]==3.8.4
```

Comment:
> aiohttp[speedups]==3.8.4 
aiohttp[speedups]>=3.8.4

---

unless we want to manually do updates every major release

Reply:
We have tied to specific versions as leaving it open has broken the script in the past